<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSetter</class>
 <widget class="QWidget" name="FilterSetter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilterSetter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="radiusSlider">
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="radiusInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="filterType"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
